### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by scharlottej13**
- Add new paths for taxi data
- update coiled login, setup instructions
- add coiled-runtime to environment.yml

### CODE REVIEW

**Comment by scharlottej13** [comment]
@jrbourbeau would you be the right person to review?

cc @dchudz

**Comment by dchudz** [comment]
@scharlottej13 any context on the switch to coiled-runtime? seems reasonable, just asking since that's the piece of this I'm least confident in having platform team review

If we don't hear from @jrbourbeau by tomorrow maybe just have one of us (e.g. me) review

**Comment by scharlottej13** [comment]
> any context on the switch to coiled-runtime?

@dchudz ah, yup good point! I made this switch for two reasons 1) to make sure "good" Dask/Distributed versions were being used and 2) to make starting a Coiled cluster less prone to package mismatches.

I started by updating the Dask/Distributed versions and then realized once you get to making a Coiled cluster the smoothest options are either using `package_sync` or using coiled-runtime. Using coiled-runtime seemed like a good choice for addressing both 1 + 2.

**Review by jrbourbeau — COMMENTED** [review]
Thanks @scharlottej13. Apologies for the delayed response, I've not had a chance to look at this yet. @ncclementi maybe this is easy for you to review? Regardless, I'll put this on our board and will bring it up at our standup tomorrow to find someone to review

**Comment by scharlottej13** [comment]
> Thanks @scharlottej13. Apologies for the delayed response, I've not had a chance to look at this yet. @ncclementi maybe this is easy for you to review? Regardless, I'll put this on our board and will bring it up at our standup tomorrow to find someone to review

Sounds good, thanks for your help! In the future, if it's helpful for me to add things to your project board instead of pinging you all the time, I'm happy to do that.

File: environment.yml [hunk 1]
```diff
@@ -2,24 +2,5 @@ name: talkpython-dask
...
-  - bokeh>=2.3.0
-  - pip>=20.3.0
-  - pip:
-    - dask-labextension>=3.0.0
-    - coiled
-  - python-graphviz
-  - h5py
-  - mimesis
+  - python=3.9
+  - coiled-runtime>=0.2.0
```
Comment (ncclementi) [inline environment.yml]: let's do >= 0.2.1 (since this one has the latest pyarrow)
```suggestion
  - coiled-runtime>=0.2.1
```

**Review by ncclementi — APPROVED** [review]
Let's wait until CI passes here https://github.com/coiled/coiled-runtime/pull/617 and this gets merged. This will enable the 0.2.1 runtime soft env, and then we can use it.

**Comment by mikeckennedy** [comment]
Hi all, thanks for the update. Given you all are from Coiled you are the experts here, I'll follow your lead. Just let me know when it's ready to be merged.

**Comment by ncclementi** [comment]
@scharlottej13 @mikeckennedy things are ready!, Whoever sees this first feel free to merge. 🎉

**Comment by mikeckennedy** [comment]
Thanks a bunch!

**Pull request merged**